src/System.Private.CoreLib/shared/System/PinnableBufferCache.cs

```diff
@@ -89,6 +89,7 @@ internal object Allocate()
             if (!m_FreeList.TryPop(out returnBuffer))
                 Restock(out returnBuffer);
 
+#if CORECLR || LOGGING
```

Comment:
I do not see any problem with enabling this for CoreRT. Is there anything that prevents that?

Reply:
Currently there is no ```GetEnumerator``` function in corert. we will have to add GetEnumerator to corert.